controllers/servers/csi/volume_group_server.py

```diff
@@ -81,12 +82,18 @@ def DeleteVolumeGroup(self, request, _):
     def _add_volumes_missing_from_group(self, array_mediator, volume_ids_in_request, volume_ids_in_volume_group,
                                         volume_group_id):
         for volume_id in volume_ids_in_request:
-            if volume_id not in volume_ids_in_volume_group:
+            if volume_id not in volume_ids_in_volume_group \
+                    and convert_scsi_id_to_nguid(volume_id) not in volume_ids_in_volume_group:
```

Comment:
IMO it can be clearer in `_is_volume_id_in_volume_group` function (but you can open a ticket because of GA)

Reply:
done